src/m365/spo/commands/page/page-list.mock.ts

```diff
@@ -333,7 +333,6 @@ export const fileApiListData = [
       "OData__SPTranslationLanguage": null,
       "OData__SPTranslationSourceItemId": null,
       "OData__SPIsTranslation": null,
-      "ID": 297,
```

Comment:
Let's revert this.

Reply:
Hi @milanholemans, I removed it because the mock was used as stub response and log output in the spec file. I didn't want to duplicate it. Apparently I overlooked something if you were able to revert it without tests breaking